graviti/manager/commit.py

```diff
@@ -136,9 +136,23 @@ def search(self, sheet: str, criteria: Dict[str, Any]) -> DataFrame:
             The created :class:`~graviti.dataframe.DataFrame` instance.
 
         """
+        total_count = create_search(
```

Comment:
Can we wrap `create_search` and common arguments into a method like `_create_search`?